xgi/stats/__init__.py

```diff
@@ -480,14 +480,6 @@ def nodestat_func(func):
     NodeStat('degree')
 
     However, `my_degree` is not recognized as a node statistic.
-
-    >>> H.my_degree()
-    Traceback (most recent call last):
-    AttributeError: stat "my_degree" not among available node or edge stats
-    >>> H.nodes.my_degree
-    Traceback (most recent call last):
-    AttributeError: Stat 'my_degree' not defined
-
```

Comment:
Do we really just want to remove this part of the docstring instead of fixing it? If we just remove this chunk, now the surrounding test doesn't really make sense.

Reply:
This should be fixed but just waiting for Github Actions to finish to double check.

Reply:
Suuuper weird this happened, I don't see why this would happen. In the interest of merging this quickly, why don't we just
```python3
Traceback (most recent call last):
AttributeError:...
```

Reply:
Okay, this fixed it! Merging now.